src/dir.targets

```diff
@@ -23,5 +23,8 @@
   <Target Name="DumpTargets" BeforeTargets="ResolveProjectReferences">
     <Message Text="DumpTargets> $(OutputPath), C=[$(Configuration)], CG=[$(ConfigurationGroup)], OG=[$(OSGroup)], TG=[$(TargetGroup)]" Importance="Low" />
   </Target>
+  
+  <!-- Workaround for issue seen after build tools update, some test projects mysteriously lose this target despite not needing it --> 
+  <Target Name="GetTargetFrameworkProperties"/>
```

Comment:
I spent a lot of time trying to track this down, since really nothing should work if the .targets that define this were missing (Tools/Microsoft.Common.CurrentVersion.targets) but this works for build, test, and packaging.  We should definitely figure it out at some point.